crates/storage/src/collections/hashmap/mod.rs

```diff
@@ -502,7 +502,7 @@ where
         self.values_entry.into_key()
     }
 
-    /// Sets the value of the entry with the `VacantEntry`'s key, and returns a mutable reference to it.
+    /// Sets the value of the entry with the `VacantEntry`s key, and returns a mutable reference to it.
```

Comment:
actually before was more correct imo ...

Reply:
Maybe @drahnr can shine some light on it, but I get a spellchecking mistake otherwise. I think this might be due to https://github.com/drahnr/cargo-spellcheck/issues/158.

But in general I agree with you, the correct version would be with `'s`.

Reply:
It's indeed the ref'd issue.

Reply:
but I thought our CI disables NLP?

Reply:
It's hunspell which complains, my understanding is that this is because the pre-processing (?) of `cargo-spellcheck` ignores terms framed in backticks.

Reply:
It's indeed the same issues as #158 . For now you could just ignore `'s` as part of the `quirks` cfg.